ifbcat_api/model/tool/tool.py

```diff
@@ -155,6 +147,13 @@ def update_information_from_json(self, tool: dict):
         # software_version = tool['version']
         # downloads = tool['download']
         self.tool_license = tool['license']
+        for doc in tool['documentation']:
```

Comment:
Have to be rework, this algorithm does not work the way it should after https://github.com/IFB-ElixirFr/ifbcat/issues/181#issuecomment-1017261102

Reply:
good improvment, but still not working 100%:

if FAQ is the first doc, and General the second, FAQ url will be used